pkg/ssh/ssh.go

```diff
@@ -82,6 +120,50 @@ func execSSHCommand(provider *SSHProvider, command string, output io.Writer) err
 	return cmd.Run()
 }
 
+func copyCommandToRemote(provider *SSHProvider, command string) (string, error) {
+	script, err := os.CreateTemp("/tmp/", "devpod-command-*")
+	if err != nil {
+		return "", err
+	}
+	defer func() {
+		script.Close()
+		os.Remove(script.Name())
+	}()
+
+	commandToRun, err := getSCPCommand(provider, script.Name())
+	if err != nil {
+		return "", err
+	}
+
+	_, err = script.WriteString(command)
+	if err != nil {
+		return "", err
+	}
+
+	return script.Name(), exec.Command("scp", commandToRun...).Run()
```

Comment:
don't think we can use `scp` here, it's not installed by default on windows machines